src/control/controlobjectscript.h

```diff
@@ -14,7 +14,7 @@ class ControlObjectScript : public ControlProxy {
             const RuntimeLoggingCategory& logger,
             QObject* pParent = nullptr);
 
-    bool addScriptConnection(const ScriptConnection& conn);
+    bool addScriptConnection(ScriptConnection* const conn);
```

Comment:
Please remove the `const` keyword. It has no effect apart from changing the function signature -> C++ Core Guidelines

Reply:
With the std::unique_ptr I could changed this backed to the original pass by reference.